<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayBar</class>
 <widget class="QWidget" name="PlayBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>39</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="playPauseButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PlaySlider" name="progressSlide">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="drawButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Start playback</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionMeasure_distance">
   <property name="icon">
    <iconset resource="../cvmob_resource.qrc">
     <normaloff>:/images/icons/draw-distance.png</normaloff>:/images/icons/draw-distance.png</iconset>
   </property>
   <property name="text">
    <string>Measure distance</string>
   </property>
   <property name="toolTip">
    <string>Measure the distance between two points</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCalculate_trajectory">
   <property name="icon">
    <iconset resource="../cvmob_resource.qrc">
     <normaloff>:/images/icons/trajectory.png</normaloff>:/images/icons/trajectory.png</iconset>
   </property>
   <property name="text">
    <string>Calculate trajectory</string>
   </property>
   <property name="toolTip">
    <string>Calculate the trajectory of the selected point in the video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionTrack_angle">
   <property name="icon">
    <iconset resource="../cvmob_resource.qrc">
     <normaloff>:/images/icons/angle.png</normaloff>:/images/icons/angle.png</iconset>
   </property>
   <property name="text">
    <string>Track angle</string>
   </property>
   <property name="toolTip">
    <string>Track the changes of an angle over time</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaySlider</class>
   <extends>QSlider</extends>
   <header location="global">view/playslider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../cvmob_resource.qrc"/>
 </resources>
 <connections/>
</ui>
